src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -78,8 +81,8 @@ object DepositHandler {
       case FailedDepositException(_, msg, cause) =>
         log.error(s"[$id] Failed deposit", cause)
         DepositProperties.set(id, "FAILED", msg, lookInTempFirst = true)
-      case cause: Throwable =>
-        log.error(s"[$id] Unexpected failure in deposit", cause)
+      case _: Throwable =>
```

Comment:
Removing the `cause` was an accident.  Now I remember that I wanted to test something in the behavior of showing the error message and had then forgotten to cancel the change.